compiler/testData/diagnostics/testsWithJvmBackend/contextReceivers/kt59590.fir.kt

```diff
@@ -0,0 +1,8 @@
+// WITH_STDLIB
+// !RENDER_ALL_DIAGNOSTICS_FULL_TEXT
+
+<!CONFLICTING_JVM_DECLARATIONS!>object O {
+    <!CONFLICTING_JVM_DECLARATIONS!>context(O)
```

Comment:
You mean to remove `context(O)` ? I prefer to keep it to show the JVM signature with a context receiver in diagnostics, since our changes have an impact on it.

Reply:
Yes. Context receivers are an experimental feature, and it won't be released as stable for quite some time. The main problem in the linked issue is that code using just stable Kotlin fails because of something related to an unstable feature. We need to have a regression test so that it won't happen again.

Feel free to duplicate this test to another file and use context receivers there. Probably it'll be best to test both with and without context receivers.

Reply:
Updated.